oxide-auth/src/code_grant/refresh.rs

```diff
@@ -237,7 +237,7 @@ impl Refresh {
     ///
     /// The provided `Input` needs to fulfill the *previous* `Output` request. See their
     /// documentation for more information.
-    pub fn advance<'req>(&mut self, input: Input<'req>) -> Output<'_> {
+    pub fn advance(&mut self, input: Input) -> Output { // Breaking change for 0.6
```

Comment:
Not actually breaking, I think. This should be the same lifetimes as implied, no?